examples/full-jaffle-shop-demo/dbt/models/customers.sql

```diff
@@ -59,6 +59,7 @@ final as (
         customers.customer_id,
         customers.first_name,
         customers.last_name,
+        30 as age, -- fixed age is filtered using required_attributes on schema.yml
```

Comment:
added for testing, but we can keep it there, I've added a new "fixed" field to not break anything else, let me know if you want me to remove it or  replace it